src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -42,6 +43,11 @@ def create(input_dataset, annotations=None, num_epochs=100, feature="bitmap",
     
     start_time = _time.time()
 
+    if max_iterations == 0:
+        max_iterations = num_epochs * len(input_dataset) / batch_size
+    else:
+        num_epochs = max_iterations * batch_size / len(input_dataset)
```

Comment:
Why should we expose both `num_epochs` and `max_iterations` as parameters?

Reply:
Tracked in #1488